src/skadi/sample.cc

```diff
@@ -470,29 +476,18 @@ template <class coord_t> double sample::get(const coord_t& coord) {
   return tile.get(u, v);
 }
 
+template <class coord_t> double sample::get(const coord_t& coord) {
+  tile_data tile;
+  return get(coord, tile);
+}
+
 template <class coords_t> std::vector<double> sample::get_all(const coords_t& coords) {
   std::vector<double> values;
   values.reserve(coords.size());
-  uint16_t curIndex = TILE_COUNT;
```

Comment:
`tile_data` already stores index so I just added `tile_data::get_index` getter to obtain its index